packages/ui/Marquee/Marquee.module.less

```diff
@@ -27,6 +27,7 @@
 		transition-property: left, right;
 		transition-timing-function: linear;
 		position: relative;
+		text-indent: 0.05em;  // Fix for characters like `j` being left-truncated because of being too close to the edge
```

Comment:
0.05em looks also cut
![image](https://user-images.githubusercontent.com/4288375/76290861-53bfb580-62ef-11ea-9628-87db11cbe98d.png)

Reply:
I have confirmed this too. The existing 0.05 em is not enough. I think this figure is 0.06 em is appropriate.